api/instance/v1/server_utils_test.go

```diff
@@ -13,6 +14,14 @@ import (
 	"github.com/scaleway/scaleway-sdk-go/scw"
 )
 
+func init() {
+	// set interval strategy to 0 when replaying cassettes
+	_, UpdateCassette := os.LookupEnv("UPDATE")
```

Comment:
shouldn't we use "UPDATE_GOLDEN" ?

Reply:
This is cassette, so it would be `UPDATE_CASSETTE`